core/pta/device.c

```diff
@@ -82,13 +83,21 @@ static TEE_Result invoke_command(void *pSessionContext __unused,
 				 uint32_t nCommandID, uint32_t nParamTypes,
 				 TEE_Param pParams[TEE_NUM_PARAMS])
 {
+	TEE_Result res = TEE_SUCCESS;
+
```

Comment:
before that, a check needs to be added in add_ta().
That function already treats TA_FLAG_DEVICE_ENUM and TA_FLAG_DEVICE_ENUM_SUPP as mutually exclusive.
The new RPMB flag has to be checked as well